Documentation/ConfigFile.md

```diff
@@ -14,28 +14,29 @@ The configuration file is a YAML file structured like this (example):
 input_dir: Sources/Resources
 output_dir: Sources/Generated/
 strings:
-  inputs: Base.lproj/Localizable.strings
+  inputs: Base.lproj
```

Comment:
I honestly didn't notice. Pretty sure PathKit handles this correctly (inserts `/` when appending if needed).

It's not intentional, but we may as well leave it so that some users may notice that it is possible.